packages/components/pie-divider/src/divider.scss

```diff
@@ -1,22 +1,50 @@
 @use '@justeattakeaway/pie-css/scss' as p;
 
 .c-divider {
-    --divider-bg-color: var(--dt-color-divider-default);
-    --divider-width: 100%;
-    --divider-height: 1px;
+  --divider-bg-color: var(--dt-color-divider-default);
+  --divider-width: 100%;
+  --divider-height: 1px;
+  --divider-lebel-max-width: 90%;
 
-    margin: 0;
-    border: 0;
-    width: var(--divider-width);
-    height: var(--divider-height);
-    background-color: var(--divider-bg-color);
+  margin: 0;
+  border: 0;
+  width: var(--divider-width);
+  height: var(--divider-height);
+  background-color: var(--divider-bg-color);
 
-    &.c-divider--inverse {
-        --divider-bg-color: var(--dt-color-divider-inverse);
+  &.c-divider--labelled {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    height: auto;
```

Comment:
I think it's a little cleaner to avoid having to undo/reset CSS properties like this. 

Could we target the c-divider class when it does not have the labelled modifier and apply the 1px height there?

[Example](https://sass-lang.com/playground/#eJwzNNRL1k3JLMtMSS1SqOZSULDKyy/RQIjp6uYkJqXm5KSmaIKlFRQyUjPTM0qsFAwLKqyBArVADERq2LRAdSQlJmenF+WX5qVYKRSlpkB01QIAzDckYQ==)